README.md

```diff
@@ -129,6 +129,8 @@ Browse Admin Interface
 
 http://localhost:3000/admin
 
+You can generate a new admin account by running `rake spree_auth:admin:create`.
```

Comment:
could you add some info that you need `spree_auth_devise` installed if you want to use it?